analysis_options.yaml

```diff
@@ -1,18 +1,12 @@
+include: package:lint/analysis_options_package.yaml
+
 analyzer:
   strong-mode:
-    implicit-casts: false
     implicit-dynamic: false
 
 linter:
   rules:
-    - await_only_futures
-    - cancel_subscriptions
-    - close_sinks
-    - hash_and_equals
-    - iterable_contains_unrelated_type
-    - list_remove_unrelated_type
-    - sort_constructors_first
-    - test_types_in_equals
-    - unnecessary_new
-    - unrelated_type_equality_checks
-    - valid_regexps
+    # const is automatically added by the Dart VM
+    prefer_const_constructors: false
```

Comment:
Very good!

Reply:
Very bad! https://github.com/passsy/dart-lint/issues/22#issuecomment-726751958

Reply:
@passsy Thanks for having a look :P

Do you think const will provide a noticeable performance benefit for flutter apps?
If not I would rather not pollute the code with const

Reply:
If you'd remove all `const` from the Flutter SDK you'd definitely notice it. A single package won't have a big impact on performance. But when we all push the usage of `const` it will make a difference.
Adding a few `const` here and there is a rather small price to pay.

Reply:
Hmm, good point having it in packages will help not that being a bottleneck in an app implementing it.